cristi-lupu/Youth
# Remove spaces from folder's names

Reviewer: cristian.lupu

### Background

Adding spaces in folder name is a bad habit (subjective opinion!).

### What has been done?
1. Removed all spaces from the folder's names (🙏🤞).
2. Updated README (because why not?).
3. Update VIPER template.

### How to test?
1. Run all the things. If seriously, just check unit tests (because they include building app + running some unit tests).

### Checklist
